client/post-editor/editor-sharing/publicize-options.jsx

```diff
@@ -204,34 +206,22 @@ const EditorSharingPublicizeOptions = React.createClass( {
 		this.props.dismissShareConfirmation( this.props.siteId, this.props.post.ID );
 	},
 	render: function() {
-		if ( ! this.props.isPublicizeEnabled ) {
+		const { isPublicizeEnabled, publicizePermanentlyDisabled } = this.props;
+		if ( ! isPublicizeEnabled ) {
 			return null;
 		}
 
-		if ( this.props.site && this.props.site.options.publicize_permanently_disabled ) {
+		if ( publicizePermanentlyDisabled ) {
 			return (
 				<div className="editor-sharing__publicize-disabled">
 					<p><span>{ this.translate( 'Publicize is disabled on this site.' ) }</span></p>
 				</div>
 			);
 		}
 
-		if ( this.props.site && this.props.site.jetpack && ! this.props.site.isModuleActive( 'publicize' ) ) {
```

Comment:
I know it's been unreachable for a while, but this render block seems much more desirable an effect than returning `null` if Publicize is not enabled. Also, if this were kept as-is, the `jetpackModulePopup` and `onModuleConnectionPopupClosed` functions would need to be removed.